student/src/main/java/tds/student/sql/repository/remote/impl/RemoteExamRepository.java

```diff
@@ -252,6 +255,28 @@ public void approveAccommodations(UUID examId, ApproveAccommodationsRequest appr
   }
 
   @Override
+  public Optional<ValidationError> reviewExam(final OpportunityInstance opportunityInstance) throws ReturnStatusException, IOException {
+    UriComponents uriComponentsBuilder = UriComponentsBuilder.fromUriString("{examUrl}/{examId}/review")
+        .queryParam("sessionId", opportunityInstance.getSessionKey())
+        .queryParam("browserId", opportunityInstance.getExamBrowserKey())
+        .buildAndExpand(examUrl, opportunityInstance.getExamId());
+
+    try {
+      restTemplate.put(uriComponentsBuilder.encode().toUri(), null);
+    } catch (HttpClientErrorException hce) {
+      // An unprocessable entity response means that a validation error was returned by the exam service.  Extract the
+      // message and return it as a ValidationError to the caller.
+      if (hce.getStatusCode().equals(HttpStatus.UNPROCESSABLE_ENTITY)) {
+        return Optional.of(objectMapper.readValue(hce.getResponseBodyAsString(), ValidationError.class));
+      }
+    } catch (RestClientException rce) {
+      throw new ReturnStatusException(rce);
```

Comment:
@wtritch's been suggesting a small tweak to the handling of the RCE, which I agree with and have been adding.  The below ensures that we can see the reason it failed.

```
     final ReturnStatusException statusException = new ReturnStatusException("Failed to find page with items: " + e.getResponseBodyAsString());
      statusException.getReturnStatus().setHttpStatusCode(500);
      throw statusException;
```